cordova-plugin-outline/android/java/org/outline/vpn/VpnTunnel.java

```diff
@@ -85,6 +85,9 @@ public synchronized boolean establishVpn() {
             vpnService.getSystemService(ConnectivityManager.class).getActiveNetwork();
         builder.setUnderlyingNetworks(new Network[] {activeNetwork});
       }
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Comment:
What happens in versions before Q?

Reply:
This method does not exist prior to Q, and the VPN is always treated the same as the specified underlying network.